docs/jwst/extract_1d/description.rst

```diff
@@ -302,3 +301,43 @@ non-differentiated error term.  Note that while covariance is also extremely imp
 (as the IFUs themselves are significantly undersampled) this term is not presently computed or taken
 into account in the ``extract_1d`` step.  As such, the error estimates should be taken as a rough
 approximation that will be characterized and improved as flight data become available. 
+
```

Comment:
Given that this is a fairly extensive new set of material - unique to MIRI MRS - I suggest putting it into its own subsection within the larger Description. Perhaps this new outline of sections/subsections:

Extraction for 3D IFU Data   (already existing section header)
    Source and Background Extraction Regions   (new subsection header at top of existing IFU section)
    MIRI MRS 1D Residual Fringe Correction   (new subsection header at the beginning of this new material)